src/client/containers/login-page/login-page.js

```diff
@@ -0,0 +1,47 @@
+import React, { useState } from 'react';
+import { useHistory } from 'react-router-dom';
+
+import './login-page.css';
+import '../../components/login/login.style.css';
+
+import Login from '../../components/login/login.component';
+
+import firebase from '../../firebase/auth';
+
+function LoginPage() {
+  const [email, setEmail] = useState('');
+  const [password, setPassword] = useState('');
+  const [errorMessage, setErrorMessage] = useState('');
+  const [errorNotifications, setErrorNotifications] = useState(null);
+
+  const history = useHistory();
+
+  return (
+    <main>
+      <Login
```

Comment:
A bit too generic here. I think you should create an email and a password component.